README.md

```diff
@@ -32,18 +31,18 @@ For additional resources, you can visit the
 
 A SmartDataCenter installation consists of two or more servers. All servers run
 [SmartOS](http://smartos.org). One server acts as the management server, the
-headnode (HN), which houses the initial set of core services that drive SDC. The
+headnode, which houses the initial set of core services that drive SDC. The
 remainder are compute nodes (CNs) which run instances (virtual machines).
 
 SDC features:
 
 - SmartOS zones provides high performance container virtualization. KVM support
   on top of zones means secure full Linux and Windows guest OS support.
-- RESTful API and CLI tooling for customer self-service
-- Complete operations portal (web GUI)
+- RESTful API and CLI tooling for customer self-service.
+- Complete operator portal (web app).
```

Comment:
These aren't sentences. I'm no grammarian, but doesn't that mean we shouldn't use a period?